tests/pkg_micropython/tests/01-run.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
```

Comment:
Does micropython have a more in-depth test suite? The reason I ask is that the changed introduced by the RIOT port seem to be quite extensive.

Reply:
Possible. That can surely be added later. This test for now just confirms that the basic building is working and that the basic interpreter can execute simple python code.